app/bundles/CoreBundle/Assets/js/1a.content.js

```diff
@@ -1469,6 +1469,65 @@ Mautic.activateLiveSearch = function (el, searchStrVar, liveCacheVar) {
     }
 };
 
+/**
+ * Filters a list based on select value
+ *
+ * @param el
+ */
+Mautic.activateListFilterSelect = function(el) {
```

Comment:
Where is this fn being used? I can't find a place.

Reply:
let me check something, wait a minute before merging

Reply:
this function was refactored into another in the previous merged PR

rebased this branch, the function ceased existing

also tested now using the UI, all good